features/fixtures/maze_runner/Assets/Scripts/MobileScenarioRunner.cs

```diff
@@ -130,6 +128,97 @@ private Configuration PreapareConfigForScenario(string scenarioName)
         switch (scenarioName)
         {
 
+            case "On Send Native Callback":
+                config.AddOnSendError((@event) => {
+
+                    
+                    @event.ApiKey = "Custom ApiKey";
+
+                    // AppWithState
+                    var app = @event.App;
+                    app.BinaryArch = "Custom BinaryArch";
+                    app.BuildUuid = "Custom BuildUuid";
+                    app.CodeBundleId = "Custom CodeBundleId";
+                    app.Id = "Custom Id";
+                    app.ReleaseStage = "Custom ReleaseStage";
+                    app.Type = "Custom Type";
+                    app.Version = "Custom Version";
+                    app.VersionCode = 999;
+                    app.Duration = TimeSpan.FromMilliseconds(1000);
+                    app.DurationInForeground = TimeSpan.FromMilliseconds(1000);
```

Comment:
Perhaps worth considering a distinct value for duration/durationInForeground (and other fields) to ensure that the setters don't affect unexpected fields

Reply:
Done 👍